lib/dialogs/keybindings/index.tsx

```diff
@@ -46,7 +46,7 @@ const Keys = ({
 export class AboutDialog extends Component<Props> {
   render() {
     const { closeDialog, isElectron, isMacApp } = this.props;
-
+    const isMac = isMacApp || navigator.userAgent.indexOf('Mac OS X');
```

Comment:
looks like this change was an improper merge-conflict resolution.
when I ran test-merges I found a few conflicts that required careful resolution so we don't wipe-out the changes made in `develop` since we forked `release/1.16`

it's probably most helpful to review the PRs in `develop` while resolving the conflicts to see what changed and how it will need to be resolved. in this case there was a utility function introduced in #2004 that should be taken into consideration here.

off the top of my head I don't remember the others but we need to be careful with this merge or we'll erase the work that @belcherj did

Reply:
Hey @dmsnell, thanks for picking this up, phew... 😅 

> looks like this change was an improper merge-conflict resolution.

I'm not sure I follow, sorry. _This is my first run at opening this kind of PRs for the release process, bare with me._

I didn't attempt any conflict resolution, just opened the PR. In fact, GitHub is showing a few conflicting files, including this one.

<img width="619" alt="Screen Shot 2020-05-05 at 11 35 34 am" src="https://user-images.githubusercontent.com/1218433/81028400-9d2e1a80-8ec4-11ea-9048-66e7530d0540.png">

I can _try_ to resolve the conflicts, but I think it's better if one of you core devs does it, right?

Reply:
> bare with me

No worries. As long as we don't rush we'll be fine.

> I can try to resolve the conflicts

We should absolutely avoid Github's tools to resolve the conflicts since they have a tendency to force-push and destroy the repo.

Typically someone adds new commits to the release branch to resolve the conflicts. Technically that violates our official release process and practically it can make for some confusing state in the repo, leading to weird divergence between `develop` and `master`.

> I think it's better if one of you core devs does it, right?

we can try. maybe we can pull a cherry-pick from the release into `develop`. this PR would ideally need to be recreated after that.